python/cuml/dask/ensemble/randomforestregressor.py

```diff
@@ -111,6 +111,12 @@ class RandomForestRegressor(BaseRandomForestModel, DelayedPredictionMixin,
     seed : int (default = None)
         Base seed for the random number generator. Unseeded by default. Does
         not currently fully guarantee the exact same results.
+    ignore_empty_partitions: Boolean (default = False)
+        Specify behavior when a worker does not recieve any data
```

Comment:
instead of "receive any data" how about "hold any data" since we're not the ones sending the data?